### PR TITLE
Update info on contributing to reference `main` branch

### PR DESCRIPTION
**Pull request by yogurtearl**
Best I can tell, the `dev_XX` branches are no longer in use.

### CODE REVIEW

**Comment by yogurtearl** [comment]
@googlebot I signed it!

**Review by nickbutcher — APPROVED** [review]

**Comment by nickbutcher** [comment]
Thanks!

**Comment by yogurtearl** [comment]
Looks like a change to only `CONTRIBUTING.md` doesn't trigger any of the github workflows.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ accept your pull requests.
 
 ## Contributing A Patch
 
-All development is done on the latest `dev_XX` branch. You should base any changes from this branch.
+All development is done on the latest `main` branch. You should base any changes from this branch.
```
Comment (JoseAlcerreca) [inline CONTRIBUTING.md:21]:
```suggestion
All development is done on the `main` branch. You should base any changes from this branch.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -30,4 +30,4 @@ All development is done on the latest `dev_XX` branch. You should base any chang
    [Kotlin Style Guide](https://android.github.io/kotlin-guides/style.html) for the
    recommended coding standards for this organization.
 1. Ensure that your code has an appropriate set of unit tests which all pass.
-1. Submit a pull request targeting the latest `dev_XX` branch.
+1. Submit a pull request targeting the latest `main` branch.
```
Comment (JoseAlcerreca) [inline CONTRIBUTING.md:33]:
```suggestion
1. Submit a pull request targeting the `main` branch.
```

**Review by JoseAlcerreca — APPROVED** [review]

**Pull request merged**